.bluemix/wizard.yml

```diff
@@ -3,32 +3,28 @@ version: '1'
 messages:
   $i18n: locales.yml
 welcome:
-  description: 'With this toolchain, you can develop and deploy an application securely into a Kubernetes cluster managed by the IBM Cloud Kubernetes Service. By default, the toolchain uses a sample Node.js \"Hello World\" app, but you can also bring your own app and link to it instead.'
+  description: 
+      $ref: "#/messages/wizard.wecome.description"
 steps:
   - name: app
-    label: Application
-    form: 
-      name: wizard_app
-      title: Application
-    validator: repo
+    title:
+      $ref: "#/messages/wizard.step.application.title"
+    form: wizard_app
     expert_mode:
```

Comment:
There's no **expert_mode** anymore ... right ?

Reply:
Yep, will change to `advanced_form`